h2o-core/src/test/java/water/AtomicTest.java

```diff
@@ -69,7 +69,7 @@ private void doLarge(Key k) {
     Random r = new Random(1234567890123456789L);
     int total = 0;
     while( total < AutoBuffer.MTU*8 ) {
-      byte[] kb = new byte[Key.KEY_LENGTH];
```

Comment:
Because the constant no longer exists in the main source code.

There is no limit on the key size anymore, it can be an arbitrary number.